src/ReturnBookUI.java

```diff
@@ -1,87 +1,81 @@
 import java.util.Scanner;
 
-
 public class ReturnBookUI {
 
-	public static enum UI_STATE { INITIALISED, READY, INSPECTING, COMPLETED };
+	public enum UIState { INITIALISED, READY, INSPECTING, COMPLETED };

```

Comment:
Removing redundant static modifiers is not established in our code guidelines.
Discuss in meeting.

Reply:
As discussed, adding to guidelines that redundant static modifiers on enums will be removed, as enums are inherently static.